client/selfrepair/normandy_driver.js

```diff
@@ -1,4 +1,3 @@
-import EventEmitter from 'wolfy87-eventemitter';
```

Comment:
I think this is the only place we use `wolfy87-eventemitter`, besides tests. We can probably get rid of the dependency.